packages/console/src/pages/ApplicationDetails/ApplicationDetailsContent/index.tsx

```diff
@@ -205,7 +205,7 @@ function ApplicationDetailsContent({ data, oidcConfig, onApplicationUpdated }: P
             {![ApplicationType.MachineToMachine, ApplicationType.Protected].includes(data.type) && (
               <RefreshTokenSettings data={data} />
             )}
-            <BackchannelLogout />
+            {data.type !== ApplicationType.MachineToMachine && <BackchannelLogout />}
```

Comment:
How about the API level guard?

Reply:
it doesn't hurt actually. just meaningless. so only frontend guard should be fine

Reply:
LOG-9190